deploy/examples/import-external-cluster.sh

```diff
@@ -34,6 +35,12 @@ function checkEnvVars() {
   if [ -z "$NAMESPACE" ]; then
     echo "Please populate the environment variable NAMESPACE"
     exit 1
+  fi
+  if [ -z "$ROOK_RBD_FEATURES" ]; then
+    echo "Please populate the environment variable ROOK_RBD_FEATURES"
+    echo "For a kernel earlier than 5.4 use a value of 'layering'; for 5.4 or later"
+    echo "use 'layering,fast-diff,image-map,deep-flatten,exclusive-lock'"
+    exit 1
```

Comment:
maybe use this check to for `layering`, since it is a requirement ?

Reply:
I'm not sure that it is actually a requirement, and we don't sanity-check any other arguments in this section.  Nonetheless I'm pushing an update.